src/CMakeLists.txt

```diff
@@ -133,6 +133,13 @@ function(valhalla_module)
   install(FILES ${MODULE_HEADERS}
     DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/${MODULE_NAME}"
     COMPONENT development)
+  
+  install(DIRECTORY ${MODULE_HEADERS_VENDORED}
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/third_party/"
+    FILES_MATCHING
+    PATTERN "*.h"
+    PATTERN "*.hpp"
+    PATTERN "include")
```

Comment:
this installs properly, but leaves a lot of empty directories in the install tree and sub-submoduled headers, see PR description